publishing-bot/configs/kubernetes-nightly-rules-configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: publisher-rules
+data:
+  config: |
```

Comment:
Unfortunately I'm not aware of how the publishing bot works, but why are we creating empty ConfigMaps? Is is somehow related for it to work with default values?

Reply:
Answered here: https://github.com/kubernetes/k8s.io/pull/520#discussion_r368515333